# frame names
frame_ids: [local_origin]

############### CNN-related parameters ###############
threshold: 0.1
hier_threshold: 0.1
only_subsquare: false

############### camera tf parameters  ###############
camera_offset_x: 0.0
camera_offset_y: -0.04
camera_offset_z: -0.2

camera_offset_roll: 45 #degrees
camera_offset_pitch: -90  #degrees
camera_offset_yaw: 45  #degrees

camera_delay: 0 #ms

############### position estimation parameters ###############
ass_thresh: 0.0     # threshold for associating detections to detected UAVs
sim_thresh: 0.001   # threshold for erasing similar detected UAVs
